src/ReCaptcha.php

```diff
@@ -17,40 +16,34 @@
  */
 class ReCaptcha
 {
-    /**
-     * URI to the regular API
-     *
-     * @var string
-     */
-    const API_SERVER = 'http://www.google.com/recaptcha/api';
 
     /**
-     * URI to the secure API
+     * URI to the API
      *
      * @var string
      */
-    const API_SECURE_SERVER = 'https://www.google.com/recaptcha/api';
```

Comment:
Hi @Maks3w. According to the recaptcha documentation at https://developers.google.com/recaptcha/docs/display under "Automatically render the reCAPTCHA widget", the script must be used only with HTTPS.